### PR TITLE
Update action to exit when msg is empty

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
if Text or Attachments are empty exit without failing 

first part to fix https://github.com/mattermost/mattermost-developer-documentation/pull/496#issuecomment-574144400

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
 
-go 1.12
+go 1.13
+
+require (
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200114132950-e77ef9733c97
+	github.com/pkg/errors v0.9.0
+	go.uber.org/zap v1.13.0 // indirect
+	golang.org/x/crypto v0.0.0-20200109152110-61a87790db17 // indirect
+	gopkg.in/yaml.v2 v2.2.7 // indirect
```
Comment (hanzei) [inline go.mod:10]: How did you generate the file? Are the indirect imports needed?

File: go.mod [hunk 2]
```diff
@@ -1,3 +1,11 @@
 module github.com/cpanato/action-mattermost-notify
 
-go 1.12
+go 1.13
+
+require (
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200114132950-e77ef9733c97
```
Comment (hanzei) [inline go.mod:6]: How did you generate this line? I was expecting it to look like https://github.com/mattermost/mattermost-plugin-jira/blob/master/go.mod#L11

**Review by hanzei — APPROVED** [review]
Thanks for the quick fix :+1: 

Two comment from my side, but none is blocking

**Review by lieut-data — APPROVED** [review]
No objections beyond @hanzei's concerns.

File: go.mod (hunk 1 above)
Comment (cpanato) [reply go.mod:10]: i can build it again

**Pull request merged**